overlay/overlay_launchers.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_OVERLAY_LAUNCHERS_H_
+#define MUMBLE_OVERLAY_LAUNCHERS_H_
+
+static const char *overlayLaunchers[] = {
+	"Steam.exe", // Steam
+	"EALaunchHelper.exe", // Origin
+	"Battle.net.exe", // Battle.net
+	"GalaxyClient.exe", // GOG Galaxy
+	"UbisoftGameLauncher.exe", // Uplay
+	"UbisoftGameLauncher64.exe", // Uplay
+
```

Comment:
For Steam, the parent is Steam.exe:

![image](https://cloud.githubusercontent.com/assets/36527/23826893/328e2e2a-06a7-11e7-8753-98dee2edb163.png)

But as you've commented below for Origin, that isn't/wasn't the case.